pom.xml

```diff
@@ -22,10 +22,10 @@
         <groupId>net.openhft</groupId>
         <artifactId>java-parent-pom</artifactId>
         <version>1.1.33</version>
-        <relativePath />
+        <relativePath></relativePath>
     </parent>
     <artifactId>chronicle-core</artifactId>
-    <version>2.23ea14-SNAPSHOT</version>
+    <version>2.23ea20-SNAPSHOT</version>
```

Comment:
need to rebase?

Reply:
This is to ensure I am using the version I think I am, as this is a significant change that took a while to get right.